plugins/cryptolib/cryptolib.cpp

```diff
@@ -411,6 +411,52 @@ void verifyPKIAlgorithm(const char * pkAlgorithm)
 }
 
 
+//---------------------------------------------------------
+// Helper that reads the given logical file into a string buffer
+// Throws if user does not have read permission or if FNF
+//---------------------------------------------------------
+void loadLFS(const char * lfs, IUserDescriptor * user, StringBuffer &sb)
+{
+    CDfsLogicalFileName lfn;
+    lfn.set(lfs);
+    try
+    {
+        Owned<IDistributedFile> df = queryDistributedFileDirectory().lookup(lfn, user, false, false, false, nullptr, defaultPrivilegedUser);//scope checks
+        if (!df)
+        {
+            throw MakeStringException(-1, "File Not Found");
```

Comment:
minor: preferable to use makeStringExceptioV and output lfn name in exception message.